brokers/unified/broker.go

```diff
@@ -137,42 +142,84 @@ func (b *Broker) ProcessPlugins(metas []model.PluginMeta) error {
 	return nil
 }
 
+func ValidateMeta(meta model.PluginMeta) error {
+	switch meta.APIVersion {
+	case "":
+		return fmt.Errorf("Plugin '%s' is invalid. Field 'apiVersion' must be present", meta.ID)
```

Comment:
Minor style detail: error messages should start with a lowercase letter, per https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Well, this is just the same error message as was before my change. I just moved it during the `Extract Function` refactoring.

Reply:
If it's in the diff already, it may as well be improved.

Reply:
Sure, but then shouldn't I change it in all error messages then ? Shouldn't it be preferably done in a distinct PR ?